include/TPP/Dialect/VNNI/VNNIOps.td

```diff
@@ -0,0 +1,33 @@
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef CHECK_DIALECT_CHECK_OPS
+#define CHECK_DIALECT_CHECK_OPS
+
+include "mlir/IR/OpBase.td"
+
+def VNNI_Dialect : Dialect {
+  let name = "vnni";
+  let cppNamespace = "::mlir::vnni";
+  let summary = [{
+    A dialect implementing vnni operations.
+  }];
+}
+
+def VNNI_MatmulOp : Op<VNNI_Dialect, "matmul"> {
+  let summary = [{Matmul operation with VNNI format of the first operand}];
+  let description = [{
+    Matmul Operation with VNNI format of the first operand..
```

Comment:
The description read the same as the summary. Can you please improve? For example, can we add the access pattern of the operation (i.e., C[i][j] += A[i][k] * B[k][j]).